slang-checks/src/main/java/org/sonarsource/slang/checks/ElseIfWithoutElseCheck.java

```diff
@@ -19,19 +19,19 @@
  */
 package org.sonarsource.slang.checks;
 
+import org.sonar.check.Rule;
 import org.sonarsource.slang.api.IfTree;
 import org.sonarsource.slang.api.TextRange;
 import org.sonarsource.slang.api.Token;
 import org.sonarsource.slang.api.Tree;
 import org.sonarsource.slang.checks.api.InitContext;
 import org.sonarsource.slang.checks.api.SlangCheck;
 import org.sonarsource.slang.impl.TextRangeImpl;
-import org.sonar.check.Rule;
 
 @Rule(key = "S126")
 public class ElseIfWithoutElseCheck implements SlangCheck {
 
-  private static final String MESSAGE_TEMPLATE = "Add the missing \"%s\" clause.";
+  private static final String MESSAGE = "Add the missing \"else\" clause.";
```

Comment:
why?

Reply:
As discussed yesterday, ruby has an "elsif" keyword, so using this keyword for the message would be confusing, as the rule is about the final "else" clause